examples/src/main/java/io/kubernetes/client/examples/OutOfClusterClientExample.java

```diff
@@ -0,0 +1,45 @@
+/** Alipay.com Inc. Copyright (c) 2004-2019 All Rights Reserved. */
```

Comment:
Remove and add the Kubernetes copyright header.

Reply:
oh dang, my IDE tooling always leaves me a surprise..